trace-protobuf/build.rs

```diff
@@ -63,6 +63,10 @@ fn generate_protobuf() {
         ".pb.Span.spanLinks",
         "#[serde(skip_serializing_if = \"::prost::alloc::vec::Vec::is_empty\")]",
     );
+    config.field_attribute(
+        ".pb.Span.error",
+        "#[serde(skip_serializing_if = \"is_default\")]",
```

Comment:
You can use ::i32::is_zero instead.

Reply:
It fails with 
```
error[E0425]: cannot find function `is_zero` in module `std::i32`
   --> libdatadog/trace-protobuf/src/pb.rs:115:35
    |
115 |     #[serde(skip_serializing_if = "::std::i32::is_zero")]
    |      
```

Reply:
Hm. Well, try `i32::is_zero` without the std stuff.

Reply:
Same.

At first try, I used `is_negative` (https://moshg.github.io/rust-std-ja/std/primitive.i32.html#method.is_negative), but it gave me an other error:
```
error[E0308]: mismatched types
   --> trace-protobuf/src/pb.rs:57:23
    |
57  | #[derive(Deserialize, Serialize)]
    |                       ^^^^^^^^^ expected `i32`, found `&i32`
...
114 |     #[serde(skip_serializing_if = "i32::is_negative")]
    |                                   ------------------ arguments to this function are incorrect
```